proto/palomachain/paloma/evm/params.proto

```diff
@@ -7,6 +7,5 @@ option go_package = "github.com/palomachain/paloma/x/evm/types";
 
 // Params defines the parameters for the module.
 message Params {
-  option (gogoproto.goproto_stringer) = false;
```

Comment:
Why remove this?

Reply:
@byte-bandit  i think in v0.50 they have removed it .

Reply:
ok

Reply:
> Looks like it's still used in the base repo https://github.com/cosmos/gogoproto
 
in upgradation docs [v0.50.1 upgrade](https://github.com/cosmos/cosmos-sdk/blob/v0.50.1/UPGRADING.md#stringer), its mentioned so.  Should we retain it or keep it deleted? @byte-bandit

Reply:
Safe to remove then I'd say.